css/style.comp.css

```diff
@@ -0,0 +1,4 @@
+.test {
```

Comment:
Please  remove this and all files that are generate by tools.
- If you are using SASS than only SAS files (the one under sass/folder) need to be in the repository.
- Also create a .gitignore file to avoid inclusion of generated file.
- - Also remove the package-lock.json from the repo and add it to the .gitignore

Reply:
I did all the corrections and it is fixed

Reply:
Nice, I just check and is ok now. Thanks.